templates/vue/src/components/Lightbox/media/VideoMedia/YouTubeMedia.vue

```diff
@@ -256,4 +256,20 @@ export default {
   max-height: calc(100vh - 120px);
   height: 100%;
 }
+.video-title {
+  text-align: left;
+  margin-bottom: 0.9em 0 0.5em 25px;
+  font-weight: 500;
+  font-size: 1.75rem;
+
+  :before {
+    display: none;
+  }
+}
+</style>
+
+<style>
+.embed-responsive iframe {
+  min-height: 0;
+}
```

Comment:
Do you know why these styles were added (I know you didn't write them...)? They seem pretty random considering this is a PR for Kaltura. Also, 1) `.video-title` is only used in the parent component, and 2) making the `.embed-responsive iframe` rule not scoped seems not a good idea.

Reply:
I am not sure either.

On closer inspection, the `.video-title` rule is not completely valid (see the `margin-bottom` property), and is very close to the rule in the parent commit, so maybe it was a merge issue.

The `.embed-responsive iframe` rule was added in 19b79a2ee6c0903a3c4b0bb809eef314b37870b4, but YouTube videos look the same even with it removed. Since YouTube videos are working fine on master and these changes are unrelated to the PR, I think I'll just delete them. Thanks for pointing this out.